emperor/support_files/js/abc-view-controller.js

```diff
@@ -183,5 +183,14 @@ define([
     throw Error('Not implemented');
   };
 
+  /**
+   * Writes the current settings to the active decomposition view(s).
+   * Will be called when the decomposition view is swapped out
```

Comment:
- Once parallel plots are enabled. There's a few things that need to change in the UI:

    - the "axes dropdowns" in the Axes tab should be disabled.

    - The Scale, shape and animations tab should be all disabled or hidden.

    - In the context menu:
        - The export as SVG and export to VEGA should also be disabled.

        - The option to toggle auto-rotation should also be disabled.

Reply:
Done.  Also set auto-rotation to false when switching to parallel-plot mode.